sr_robot_commander/doc/tutorial/RobotCommander.md

```diff
@@ -0,0 +1,100 @@
+# Robot Commander
+
+## Description
+
+Main purpose of the commander is to provide simplified access to [hand](HandCommander.md) or [arm](ArmCommander.md).
+It provides methods which can be used on both [hand](HandCommander.md) and [arm](ArmCommander.md).
+
+Examples of usage can be found in the package **sr_example** in files **sr_hand_examples.py** and **sr_arm_examples.py**
+
+## move_to_joint_value_target 
+
+### Description
+
+This method sets target of the robot's links and moves to it. 
```

Comment:
Indicate whether the method can take a few joints only or need to take all the joints